### ISSUE TITLE
Move labels out of placeholder text for date and time inputs

### ISSUE THREAD

**Issue by kimisgold** [omeka-s-modules/NumericDataTypes#62]
"Year", "Month", "Day" are all using placeholders or select options to label their inputs. If the user provides values, these labels disappear, causing an accessibility issue. They need their own `<label>` elements.

**Comment by nelsonamaya82** [comment]
@aviannamiller I haven't been able to reproduce the issue you mentioned in https://github.com/omeka-s-modules/NumericDataTypes/issues/62#issuecomment-5185497123. I tested in Chrome, Safari, and Firefox. Based on your screen recording, it seems like a JavaScript issue, probably caused by another module or the script didn't load. Could you double-check and let me know if it's still happening on your side? If so, please tell me what browser and version you have, and the resolution you are testing it on.

I'm not able to reproduce the alignment issue in the mobile view either. I think I fixed it on both themes a coupe of weeks ago. Please make sure you have the latest version (master branch).

**Comment by kimisgold** [comment]
Fixes for Foundation are on its `main` branch in https://github.com/omeka-s-themes/foundation/commit/c2c997a0686643b735afcf76d0f1e6c842e88a05.

**Comment by nelsonamaya82** [comment]
@aviannamiller fixes are on the `main` branch for both Freedom and Lively.

**Comment by aviannamiller** [comment]
Thanks, @kimisgold and @nelsonamaya82. All previously stated issues are resolved.

One last thing–@nelsonamaya82, I see a gap pushing the Value input to the right in the mobile view for these NDT fields, both themes:
<img width="581" height="467" alt="Two fields in advanced search form, 'Number is greater than' and 'Number is less than' Fields include a Property selector and a Value input. Value input is on the next line under Property, and a gap to the left of the input, highlighted by browser dev tools, is positioning the two inputs out of alignment." src="https://github.com/user-attachments/assets/937d7a69-245c-42d4-91c0-e915ffd452f7" />

**Comment by nelsonamaya82** [comment]
Thanks @aviannamiller! I missed that gap. It's fixed now on the `main` branch for both Freedom and Lively.

**Comment by aviannamiller** [comment]
Great, thanks @nelsonamaya82!

We should be all set here now @kimisgold.
